File: PHP/DataTim.php
<?php

class DataTim{
    private $nama = 0;
    private $asal = 0;
    private $tahun = 0;

    public function __construct($nama = 0, $asal = 0, $tahun = 0){
        $this->nama = $nama;
        $this->asal = $asal;
        $this->tahun = $tahun;
    }

    public function setNama($nama){
        $this->nama = $nama;
    }

    public function getNama(){
        return $this->nama;
    }

    public function setAsal($asal){
        $this->asal = $asal;
    }

    public function getAsal(){
        return $this->asal;
    }

    public function setTahun($tahun){
        $this->tahun = $tahun;
    }

    public function getTahun(){
        return $this->tahun;
    }
}

?>

File: Python/DataTim.py
class DataTim():
    __nama = ""
    __asal = ""
    __tahun = 0
    
    def __init__(self):
        self.__nama = ""
        self.__asal = ""
        self.__tahun = 0
        
    def setNama(self, n):
        self.__nama = n
    
    def getNama(self):
        return self.__nama
    
    def setAsal(self, a):
        self.__asal = a
    
    def getAsal(self):
        return self.__asal
    
    def setTahun(self, t):
        self.__tahun = t
    
    def getTahun(self):
        return self.__tahun
    
class DataPemain(DataTim):
    __nomor = 0
    __posisi = ""
    
    def __init__(self):
        self.__nomor = 0
        self.__posisi = ""
    
    def setNomor(self, n):
        self.__nomor = n
    
    def getNomor(self):
        return self.__nomor
    
    def setPosisi(self, p):
        self.__posisi = p
    
    def getPosisi(self):
        return self.__posisi